Remove deprecated opera support

emanlove opened the pull request:
Pulls out Opera from the tests. Also updates the expected logging as Selenium version 4.1.4 made some modifications to the debug logging. Finally this includes a fix by @Brownies for the Cookies tests. As he notes in his pull request,

> [Chrome limits cookie expiry to max 400 days](https://chromestatus.com/feature/4887741241229312) since version 104 so adjusted the expiry times in cookie tests.

Commit:
Change cookie expiry times in cookie tests

emanlove committed:
Removed Opera as it is no longer supported by Selenium

Opera support was depreciated in Selenium version 4.2.0 and was removed
in version 4.3.0. Thus we have removed it from SeleniumLibrary.

emanlove committed:
Updated status checks on log messages

Modified expected lines for log message under `Page Should Contain` test
within atest/acceptance/keywords.content_assertions.robot script. Oddly
the status checker gave this response,

Message:	Keyword 'SeleniumLibrary.Page Should Contain' (index 3)
message 14 has wrong level.

Expected: FAIL
Actual: DEBUG

Original message:
Keyword 'SeleniumLibrary.Page Should Contain' (index 2) message 7 has
wrong level.

Expected: INFO
Actual: DEBUG

Original message:
Keyword 'SeleniumLibrary.Page Should Contain' (index 1) message 7 has
wrong level.

Expected: INFO
Actual: DEBUG

Original message:
Test failed as expected.

Original message:
Page should have contained text 'non existing text' but did not.

of which it appears the issue was not wrong level but wrong line. That
might be because it take what is expected and compares with the actual
as compared to it having expert knowledge (which is what I was
expecting). Now see my expectation was wrong. Testing the correction
with this commit.

emanlove committed:
Updated status checks on log messages

Selenium version 4.1.4 changed some of its logging so these change
correct for additional logging made with these keywords.

emanlove committed:
Updated status checks on log messages

Selenium version 4.1.4 changed some of its logging so these change
correct for additional logging made with these keywords.

aaltat commented on atest/acceptance/keywords/cookies.robot:
```diff
@@ -71,12 +71,12 @@ Get Cookies As Dict When There Are None
-    Should Be Equal As Integers    ${cookie.expiry.hour}           16
-    Should Be Equal As Integers    ${cookie.expiry.minute}         21
-    Should Be Equal As Integers    ${cookie.expiry.second}         35
+    Should Be Equal As Integers    ${cookie.expiry.year}           2023
```
Because this is so near now, I would start creating it dynamically at some point. Same goes for all tests that set the expiry date, but can be done in separate PR.

emanlove replied:
Will create a pr for this.

aaltat reviewed (approved):
One small thing which also can be done in a separate PR.

emanlove committed:
Removed lingering instances of Opera browers from atest

Also removed instances of phantomJS, iPhone, Andriod. Need to review
Create WebDriver code as atest did not catch these.
